api/python/t4/packages.py

```diff
@@ -950,7 +950,6 @@ def push(self, name, dest=None, registry=None, message=None):
             A new package that points to the copied objects.
         """
         validate_package_name(name)
-        self._set_commit_message(message)
```

Comment:
Should we delete the message definition too? looks like it isn't called anywhere else